src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int NUMBER_BOUND = 100;
+
+    public Ball getRandomBall() {
+        Ball newBall = new Ball(new Random().nextInt(NUMBER_BOUND),
+                new ColorSupplier().getRandomColor());
+        return newBall;
```

Comment:
Please make Random and ColorSupplier as class variable in order not to create new object each time method is called:
```suggestion
        return new Ball(random.nextInt(NUMBER_BOUND), colorSupplier.getRandomColor());
```